query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/SchemaBuilderDsl.java

```diff
@@ -0,0 +1,583 @@
+      CqlIdentifier keyspace, CqlIdentifier aggregateName) {
+    return new DefaultCreateAggregate(keyspace, aggregateName);
+  }
+
+  /**
+   * Shortcut for {@link #createAggregate(CqlIdentifier)
+   * CreateAggregateStart(CqlIdentifier.fromCql(keyspaceName),CqlIdentifier.fromCql(aggregateName)}
+   */
+  public static CreateAggregateStart createAggregate(String aggregateName) {
+    return new DefaultCreateAggregate(CqlIdentifier.fromCql(aggregateName));
+  }
+
+  /**
+   * Shortcut for {@link #createAggregate(CqlIdentifier, CqlIdentifier)
+   * CreateAggregateStart(CqlIdentifier.fromCql(keyspace, aggregateName)}
+   */
+  public static CreateAggregateStart createAggregate(String keyspace, String aggregateName) {
+    return new DefaultCreateAggregate(
+        CqlIdentifier.fromCql(keyspace), CqlIdentifier.fromCql(aggregateName));
+  }
+
+  /**
+   * Starts an DROP AGGREGATE query. This assumes the keyspace name is already qualified for the
+   * Session or Statement.
+   */
+  public static Drop dropAggregate(CqlIdentifier aggregateName) {
+    return new DefaultDrop(aggregateName, "AGGREGATE");
+  }
+
+  /** Starts an DROP AGGREGATE query for the given aggregate name for the given keyspace name. */
+  public static Drop dropAggregate(CqlIdentifier keyspace, CqlIdentifier aggregateName) {
+    return new DefaultDrop(keyspace, aggregateName, "AGGREGATE");
+  }
+
+  /**
+   * Shortcut for {@link #dropAggregate(CqlIdentifier)
+   * dropAggregate(CqlIdentifier.fromCql(aggregateName)}.
+   */
+  public static Drop dropAggregate(String aggregateName) {
+    return new DefaultDrop(CqlIdentifier.fromCql(aggregateName), "AGGREGATE");
+  }
+
+  /**
+   * Shortcut for {@link #dropAggregate(CqlIdentifier, CqlIdentifier)
+   * dropAggregate(CqlIdentifier.fromCql(keyspace),CqlIdentifier.fromCql(aggregateName)}.
+   */
+  public static Drop dropAggregate(String keyspace, String aggregateName) {
+    return new DefaultDrop(
+        CqlIdentifier.fromCql(keyspace), CqlIdentifier.fromCql(aggregateName), "AGGREGATE");
+  }
+
+  /**
+   * Compaction options for Leveled Compaction Strategy (LCS).
+   *
+   * @see CreateTableWithOptions#withCompaction(CompactionStrategy)
+   */
+  public static LeveledCompactionStrategy leveledCompactionStrategy() {
+    return new DefaultLeveledCompactionStrategy();
+  }
+
+  /**
+   * Compaction options for Time Window Compaction Strategy (TWCS).
+   *
+   * @see CreateTableWithOptions#withCompaction(CompactionStrategy)
+   */
+  public static TimeWindowCompactionStrategy timeWindowCompactionStrategy() {
+    return new DefaultTimeWindowCompactionStrategy();
+  }
+
+  /**
+   * Shortcut for creating a user-defined {@link DataType} for use in UDT and Table builder
+   * definitions, such as {@link CreateTable#withColumn(CqlIdentifier, DataType)}.
+   */
+  public static UserDefinedType udt(CqlIdentifier name, boolean frozen) {
+    return new ShallowUserDefinedType(null, name, frozen);
+  }
```

Comment:
This can be useful for DML queries too, like in casts. Maybe we should move/duplicate it to `QueryBuilderDsl`.